awscli/customizations/s3/filegenerator.py

```diff
@@ -208,13 +207,19 @@ def list_files(self, path, dir_op):
                         for x in self.list_files(file_path, dir_op):
                             yield x
                     else:
-                        size, last_update = get_file_stat(file_path)
-                        last_update = self._validate_update_time(
-                            last_update, path)
-                        yield (
-                            file_path,
-                            {'Size': size, 'LastModified': last_update}
-                        )
+                        stats = self._safely_get_file_stats(path, file_name=name)
```

Comment:
So for the ``_safely_get_file_stats`` can we just simplify the signature to just have ``path`` and just use ``file_path`` variable when we make this call instead of ``path`` and ``name``?